### ISSUE TITLE
Possible typo in fatal logger name

### ISSUE THREAD

**Issue by dekobon** [sisyphsu/slf4go#8]
When looking at the constant names in [slf_model.go](https://github.com/go-eden/slf4go/blob/master/slf_model.go#L36), I noticed that the fatal level name has only a single "L". It is spelled as "Fata" and "Level" combined. This is confusing when compared with the names of the other levels.

```go
const (
	TraceLevel Level = iota
	DebugLevel
	InfoLevel
	WarnLevel
	ErrorLevel
	PanicLevel
	FataLevel
)
```

I suggest that another constant be added called `FatalLevel` with the same numeric value as `FataLevel`.

**Comment by sisyphsu** [comment]
OMG, this is really a huge typo... 

Thanks~

**Comment by sisyphsu** [comment]
This typo has been fixed in `v1.0.7`~
